examples/3d/ssr.rs

```diff
@@ -0,0 +1,337 @@
+//! Demonstrates screen space reflections in deferred rendering.
+
+use std::ops::Range;
+
+use bevy::{
+    color::palettes::css::{BLACK, WHITE},
+    core_pipeline::{fxaa::Fxaa, Skybox},
+    input::mouse::MouseWheel,
+    math::{vec3, vec4},
+    pbr::{
+        DefaultOpaqueRendererMethod, ExtendedMaterial, MaterialExtension,
+        ScreenSpaceReflectionsBundle, ScreenSpaceReflectionsSettings,
+    },
+    prelude::*,
+    render::{
+        render_resource::{AsBindGroup, ShaderRef, ShaderType},
+        texture::{
+            ImageAddressMode, ImageFilterMode, ImageLoaderSettings, ImageSampler,
+            ImageSamplerDescriptor,
+        },
+    },
+};
+
+// The speed of camera movement.
+const CAMERA_KEYBOARD_ZOOM_SPEED: f32 = 0.1;
```

Comment:
You can do it if you feel like it but it's not required at all to merge. (I already made a note for me to do that in a follow-up PR). 

We generally try to put things like camera controllers and debug text in a separate submodule so only the setup part that shows how to enable the feature is at the top.